schemas/user.yml

```diff
@@ -54,6 +54,10 @@
         as: real_user
         short_override: The real user (ruid). Identifies the real owner of the process.
         beta: Reusing the `user` fields in this location is currently considered beta.
+      - at: process
+        as: attested_user
+        short_override: The attested user (auid). Identifies the attested used associated with the process.
```

Comment:
same here. auid isn't something that exists in linux, so maybe not needed, unless this is a name we're using elsewhere.

Reply:
fwiw: `auid` is used in the context of Linux kernel auditing.

https://github.com/linux-audit/audit-documentation/blob/b5a76394e165b104c95c94127ac3d29eed68f927/specs/fields/field-dictionary.csv#L16

Reply:
I too would remove  "auid" - confusing.  Are we planning to populate the user.id field for the externally attested user? if not it would nice to say somewhere that when attested_user represents a k8s user, how we map the field values received by the admission controller to the user fields.